### PR TITLE
Request Spec Testing

### PR DESCRIPTION
**Pull request by Gambit142**
I carried out request spec testing that fulfilled the following:

- Added RSpec as a gem to the project.
- Created a Request spec file for all controllers in the project.
- Made sure that for each action:
  - The response status was correct.
  - The correct template was rendered.
  - The response body included correct placeholder text.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -5,6 +5,8 @@ ruby '3.0.3'
 
 gem 'rubocop', '>= 1.0', '< 2.0'
```
Comment (SajjadAhmad14) [inline Gemfile:6]: Project looks great but it's missing a professional README. A professional README file should have a project title, description, and instructions to set the project up locally.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Francis, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Emmyn5600 — APPROVED** [review]
Hi @Gambit142 

Wooow👏, Your project is complete!  There is nothing else to say other than... it's time to merge it. 👏

Congratulations! 🎉 

##  Optional suggestion

-  n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: Gemfile (hunk 1 above)
Comment (Gambit142) [reply Gemfile:6]: Thank you, I forgot to set it up.

**Comment by Gambit142** [comment]
Thank you for the feedback

**Pull request merged**